generative/maisi/scripts/utils.py

```diff
@@ -0,0 +1,85 @@
+# Copyright (c) MONAI Consortium
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#     http://www.apache.org/licenses/LICENSE-2.0
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+import copy
+from typing import Sequence
+
+import numpy as np
+import skimage
+import torch
+import torch.nn.functional as F
+from monai.apps.generation.maisi.utils.morphological_ops import dilate, erode
+from monai.config import DtypeLike, NdarrayOrTensor
+from monai.utils import (
```

Comment:
It seems the imports are not used in this script except `ensure_tuple_rep`. It may be helpful to run a flake8 check. Thanks!

Reply:
I just ran ruff. Thanks!